<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProjectDialog</class>
 <widget class="QDialog" name="ProjectDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>774</width>
    <height>734</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create a new project</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="lNewProject">
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>New Project</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lblName">
     <property name="text">
      <string>&lt;Name&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>17</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="14" column="3">
      <widget class="QLineEdit" name="lineEditBroadcastAddress">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Enter the broadcast address of your protocol in &lt;span style=&quot; font-weight:600;&quot;&gt;hex&lt;/span&gt;. If you do not know what to enter here, just leave the default.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="inputMask">
        <string/>
       </property>
       <property name="text">
        <string>ffff</string>
       </property>
      </widget>
     </item>
     <item row="10" column="0" colspan="2">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Description:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="6">
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QLabel" name="lblNewPath">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-style:italic;&quot;&gt;Note: A new directory will be created.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="4" column="4" colspan="2">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Hz</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="2">
      <widget class="QLabel" name="label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Choose a path:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLineEdit" name="lineEdit_Path">
       <property name="statusTip">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="4" colspan="2">
      <widget class="QToolButton" name="btnSelectPath">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="3" column="4" colspan="2">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Sps</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0" colspan="2">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Default sample rate:</string>
       </property>
      </widget>
     </item>
     <item row="13" column="4" colspan="2">
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>57</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="3" column="3">
      <widget class="KillerDoubleSpinBox" name="spinBoxSampleRate">
       <property name="decimals">
        <number>3</number>
       </property>
       <property name="minimum">
        <double>0.010000000000000</double>
       </property>
       <property name="maximum">
        <double>1000000000000.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.001000000000000</double>
       </property>
       <property name="value">
        <double>1000000.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="9" column="0" colspan="6">
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="4" column="3">
      <widget class="KillerDoubleSpinBox" name="spinBoxFreq">
       <property name="decimals">
        <number>3</number>
       </property>
       <property name="minimum">
        <double>0.010000000000000</double>
       </property>
       <property name="maximum">
        <double>1000000000000.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.001000000000000</double>
       </property>
       <property name="value">
        <double>433920000.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="2">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Default frequency:</string>
       </property>
      </widget>
     </item>
     <item row="11" column="4" colspan="2">
      <widget class="QToolButton" name="btnAddParticipant">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset theme="list-add">
         <normaloff>.</normaloff>.</iconset>
       </property>
      </widget>
     </item>
     <item row="8" column="0" colspan="4">
      <widget class="QLabel" name="lOpenSpectrumAnalyzer">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Tip: Open &lt;a href=&quot;open_spectrum_analyzer&quot;&gt;spectrum analyzer&lt;/a&gt; to find these values.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="textFormat">
        <enum>Qt::AutoText</enum>
       </property>
       <property name="openExternalLinks">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="11" column="0" rowspan="3" colspan="2">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Participants:</string>
       </property>
      </widget>
     </item>
     <item row="12" column="4" colspan="2">
      <widget class="QToolButton" name="btnRemoveParticipant">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset theme="list-remove">
         <normaloff>.</normaloff>.</iconset>
       </property>
      </widget>
     </item>
     <item row="10" column="3">
      <widget class="QPlainTextEdit" name="txtEdDescription"/>
     </item>
     <item row="5" column="0" colspan="2">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>Default bandwidth:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="3">
      <widget class="KillerDoubleSpinBox" name="spinBoxBandwidth">
       <property name="decimals">
        <number>3</number>
       </property>
       <property name="minimum">
        <double>1.000000000000000</double>
       </property>
       <property name="maximum">
        <double>999999999999.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.001000000000000</double>
       </property>
       <property name="value">
        <double>1000000.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="6" column="3">
      <widget class="QSpinBox" name="spinBoxGain">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="value">
        <number>20</number>
       </property>
      </widget>
     </item>
     <item row="5" column="4" colspan="2">
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>Hz</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0" colspan="2">
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>Default gain:</string>
       </property>
      </widget>
     </item>
     <item row="11" column="3" rowspan="3">
      <widget class="QTableView" name="tblParticipants">
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::ExtendedSelection</enum>
       </property>
       <attribute name="horizontalHeaderCascadingSectionResizes">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>100</number>
       </attribute>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderCascadingSectionResizes">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderStretchLastSection">
        <bool>false</bool>
       </attribute>
      </widget>
     </item>
     <item row="14" column="0" colspan="2">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Broadcast address (hex):</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btnOK">
       <property name="text">
        <string>Create new project</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>37</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KillerDoubleSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>urh.ui.KillerDoubleSpinBox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>lineEdit_Path</tabstop>
  <tabstop>btnSelectPath</tabstop>
  <tabstop>spinBoxSampleRate</tabstop>
  <tabstop>spinBoxFreq</tabstop>
  <tabstop>spinBoxBandwidth</tabstop>
  <tabstop>spinBoxGain</tabstop>
  <tabstop>txtEdDescription</tabstop>
  <tabstop>tblParticipants</tabstop>
  <tabstop>btnAddParticipant</tabstop>
  <tabstop>btnRemoveParticipant</tabstop>
  <tabstop>lineEditBroadcastAddress</tabstop>
  <tabstop>btnOK</tabstop>
  <tabstop>btnCancel</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>btnCancel</sender>
   <signal>clicked()</signal>
   <receiver>ProjectDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>498</x>
     <y>221</y>
    </hint>
    <hint type="destinationlabel">
     <x>334</x>
     <y>122</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
